src/components/ui/Button/Button.js

```diff
@@ -41,7 +42,6 @@ export const Button = forwardRef(function Button(props, ref) {
       id={id}
       // eslint-disable-next-line react/button-has-type
       type={type}
-      style={style}
```

Comment:
can you please also replace style for other comps Label, Heading, Table to keep props consistent